internal/tss/utils/transaction_service.go

```diff
@@ -0,0 +1,240 @@
+package utils
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+
+	"github.com/stellar/go/clients/horizonclient"
+	"github.com/stellar/go/txnbuild"
+	"github.com/stellar/go/xdr"
+	"github.com/stellar/wallet-backend/internal/signing"
+	"github.com/stellar/wallet-backend/internal/tss"
+	tssErr "github.com/stellar/wallet-backend/internal/tss/errors"
+)
+
+var (
+	RpcPost                 = http.Post
+	UnMarshalRPCResponse    = io.ReadAll
+	UnMarshalJSON           = parseJSONBody
+	callRPC                 = sendRPCRequest
+	UnMarshalErrorResultXdr = parseErrorResultXdr
```

Comment:
```suggestion
	RPCPost                 = http.Post
	UnmarshalRPCResponse    = io.ReadAll
	UnmarshalJSON           = parseJSONBody
	callRPC                 = sendRPCRequest
	UnmarshalErrorResultXDR = parseErrorResultXDR
```

Reply:
IMO these function aliases make it harder rather than easier to follow the code because you need to F12 twice or scroll back to the top to the file to get to the implementation

Reply:
Yeap, removed all these aliases now and sending in an HTTPClient into transactionService which I can now mock